### PR TITLE
collect gold quest

### PR DESCRIPTION
**Pull request by shantahuja**
#210

### CODE REVIEW

File: stc_game/Assets/Devdog/QuestSystemPro/Scripts/Quests/ProgressAffectors/SetQuestProgressOnTriggerObject.cs [hunk 1]
```diff
@@ -15,24 +15,27 @@ private enum Use
...
             {
                 progress.Execute();
+                
             }
 
             return false;
         }
 
-        public bool OnTriggerUnUsed(Player player)
+        public  bool OnTriggerUnUsed(Player player)
```
Comment (dixonyant) [inline stc_game/Assets/Devdog/QuestSystemPro/Scripts/Quests/ProgressAffectors/SetQuestProgressOnTriggerObject.cs:35]: its adding extra whitespace here

File: stc_game/Assets/Devdog/QuestSystemPro/Scripts/Quests/ProgressAffectors/SetQuestProgressOnTriggerObject.cs [hunk 2]
```diff
@@ -15,24 +15,27 @@ private enum Use
...
+
         public QuestProgressDecorator progress;
 
         [Header("Trigger configuration")]
         [SerializeField]
         private Use _use;
 
 
-        public bool OnTriggerUsed(Player player)
+        public  bool OnTriggerUsed(Player player)
```
Comment (dixonyant) [inline stc_game/Assets/Devdog/QuestSystemPro/Scripts/Quests/ProgressAffectors/SetQuestProgressOnTriggerObject.cs:24]: extra whitespace?

File: stc_game/Assets/Resources/DevDogDb/MainLanguageDb.asset [hunk 3]
```diff
@@ -31,6 +35,12 @@ MonoBehaviour:
     is a dirty goblin chief that is taking hold of the cavern up north! We used to
     go there to mine for materials, ","c07cd0fe-ae73-49e4-bf1b-53b622b8f941":"Please
     destroy the goblin chief!","ae42898b-e3dc-4bdc-92fa-333418df4c10":"You defeated
-    him!! thank you!"},"_localizedObjects":{}}'
+    him!! thank you!","d4c77447-31e2-4788-85dd-8562da9b5a37":"Collect 300 Gold.","3d35a618-fc65-41f8-8132-bf415d2b0714":"Collect
```
Comment (dixonyant) [inline stc_game/Assets/Resources/DevDogDb/MainLanguageDb.asset]: capital T?

**Comment by shantahuja** [comment]
these are fixed!

**Review by dixonyant — APPROVED** [review]

**Pull request merged**